### PR TITLE
add  LSF config yaml to Snakemake

### PR DESCRIPTION
**Pull request by pmb59**
This PR adds logic to the sorting hat to copy the lsf_config yaml to the working directory

### CODE REVIEW

File: Snakefile-sorting-hat [hunk 1]
```diff
@@ -219,7 +219,7 @@ rule produce_reprocess_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-        cp {params.lsf_config} {input.working_directory}
+        cp {params.lsf_config} {wildcards.accession}/lsf.yaml
```
Comment (pcm32) [inline Snakefile-sorting-hat]: I would make this optional, so that it happens only if that params.lsf_config is defined.

Comment (pcm32) [reply Snakefile-sorting-hat]: Why not just link the file, then if you need to edit hosts in the middle of the run it can be done on a single place?

**Review by pcm32 — APPROVED** [review]
A few minor suggestions. Feel free to merge after considering them.

File: Snakefile-sorting-hat (hunk 1 above)
Comment (pmb59) [reply Snakefile-sorting-hat]: @pcm32  I have implemented both suggestions. Tested and it works fine

**Pull request merged**